<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PJPChanger</class>
 <widget class="QMainWindow" name="PJPChanger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>372</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PJP Changer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btOpen">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edFile">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbHCF">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Select Salesman HCF</string>
    </property>
    <widget class="QComboBox" name="cbHCF">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>23</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentText">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="ckHCF">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>285</x>
       <y>25</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>HCF</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbPC">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Select Salesman PC</string>
    </property>
    <widget class="QComboBox" name="cbPC">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>23</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>341</width>
       <height>233</height>
      </size>
     </property>
    </widget>
    <widget class="QCheckBox" name="ckPC">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>285</x>
       <y>25</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>PC</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btSave">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>325</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Changes</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbPath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbDC">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>179</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Select Salesman xx1</string>
    </property>
    <widget class="QComboBox" name="cbDC">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>23</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>341</width>
       <height>233</height>
      </size>
     </property>
    </widget>
    <widget class="QCheckBox" name="ckDC">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>285</x>
       <y>25</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>xx1</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbXC">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Select Salesman xx2</string>
    </property>
    <widget class="QComboBox" name="cbXC">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>23</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>341</width>
       <height>233</height>
      </size>
     </property>
    </widget>
    <widget class="QCheckBox" name="ckXC">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>285</x>
       <y>25</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>xx2</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="versionlabel">
    <property name="geometry">
     <rect>
      <x>333</x>
      <y>354</y>
      <width>27</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #a5a5a5;</string>
    </property>
    <property name="text">
     <string>v.2.0</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>362</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../toolsPJP/res_rc.qrc"/>
  <include location="res_rc.qrc"/>
 </resources>
 <connections/>
</ui>
